src/amo/components/LandingPage.js

```diff
@@ -5,36 +5,50 @@ import { asyncConnect } from 'redux-connect';
 import { connect } from 'react-redux';
 
 import LandingAddonsCard from 'amo/components/LandingAddonsCard';
+import NotFound from 'amo/components/NotFound';
 import { loadLandingAddons } from 'amo/utils';
 import {
   ADDON_TYPE_EXTENSION,
   ADDON_TYPE_THEME,
   SEARCH_SORT_POPULAR,
   SEARCH_SORT_TOP_RATED,
 } from 'core/constants';
-import { apiAddonType, visibleAddonType } from 'core/utils';
+import { AddonTypeNotFound } from 'core/errors';
+import log from 'core/logger';
+import {
+  apiAddonType,
+  visibleAddonType as _visibleAddonType,
```

Comment:
Indeed, that works for me. Because we use `visibleAddonType` as a variable this got messy 😒